src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static Task Timebox(this IEnumerable<Task> tasks, TimeSpan timeoutAfter,
 
                     if (t.IsFaulted && t.Exception != null)
                     {
-                        tcs.TrySetException(t.Exception.InnerException);
+                        tcs.TrySetException(t.Exception.GetBaseException());
```

Comment:
> If inner exception is again an aggregate this call walks to the exception and unpacks it

thanks for explaining. I was mostly struggling with the xml doc on the method which states:

> Returns the <see cref="System.AggregateException"/> that is the root cause of this exception.

so it seems to just unpack nested aggregateexceptions?